src/layout/Checkboxes/CheckboxesContainerComponent.test.tsx

```diff
@@ -349,13 +349,26 @@ describe('CheckboxContainerComponent', () => {
         source: {
           group: 'someGroup',
           label: 'option.from.rep.group.label',
+          description: 'option.from.rep.group.description',
+          helpText: 'option.from.rep.group.helpText',
           value: 'someGroup[{0}].valueField',
         },
       },
     });
 
     expect(getCheckbox({ name: 'The value from the group is: Label for first' })).toBeInTheDocument();
     expect(getCheckbox({ name: 'The value from the group is: Label for second' })).toBeInTheDocument();
+    expect(screen.getByText('Description: The value from the group is: Label for first')).toBeInTheDocument();
+    expect(screen.getByText('Description: The value from the group is: Label for second')).toBeInTheDocument();
+
+    await act(() =>
+      user.click(screen.getByRole('button', { name: 'Help text for The value from the group is: Label for first' })),
+    );
+    expect(screen.getByText('Help Text: The value from the group is: Label for first')).toBeInTheDocument();
+    await act(() =>
+      user.click(screen.getByRole('button', { name: 'Help text for The value from the group is: Label for second' })),
+    );
+    expect(screen.getByText('Help Text: The value from the group is: Label for second')).toBeInTheDocument();
```

Comment:
Nice tests! Love to see that you are using getByRole. 👏 React Testing Library recommends not using act() in this use-cases, I think you should be able to do something like this instead. 🤔 

```suggestion
 await user.click(screen.getByRole('button', { name: 'Help text for The value from the group is: Label for first' }))
 expect(screen.getByText('Help Text: The value from the group is: Label for first')).toBeInTheDocument();
  
 await user.click(screen.getByRole('button', { name: 'Help text for The value from the group is: Label for second' }));
 expect(screen.getByText('Help Text: The value from the group is: Label for second')).toBeInTheDocument();
```